65.md

```diff
@@ -6,71 +6,53 @@ Relay List Metadata
 
 `draft` `optional` `author:mikedilger`
 
-A special replaceable event meaning "Relay List Metadata" is defined as an event with kind `10002` having a list of `r` tags, one for each relay the author uses to either read or write to.
+Defines a replaceable event using `kind:10002` to advertise a user's preferred relays to others. 
 
-The primary purpose of this relay list is to advertise to others, not for configuring one's client.
+The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. If the marker is omitted, it means the author uses the relay for both purposes.
 
-The `content` is not used and SHOULD be an empty string.
+The `content` is not used.
 
-The `r` tags can have a second parameter as either `read` or `write`. If it is omitted, it means the author uses the relay for both purposes.
-
-Clients SHOULD, as with all replaceable events, use only the most recent kind-10002 event they can find.
-
-### The meaning of read and write
+```json
+{
+  "kind": 10002,
+  "tags": [
+    ["r", "wss://alicerelay.example.com"],
+    ["r", "wss://brando-relay.com"],
+    ["r", "wss://expensive-relay.example2.com", "write"],
+    ["r", "wss://nostr-relay.example.com", "read"],
+  ],
+  "content": "",
+  ...other fields
+```
 
-Write relays are for events that are intended for anybody (e.g. your followers). Read relays are for events that address a particular person.
+Clients must download all `kind:10002`s from their user's contact list and assemble a relay pool from there.
```

Comment:
I would say something more like:

Clients SHOULD seek the `kind:10002` event for people their user's contact list in order to know where to find their events and where to copy them on events tagging them.

But I'm also ok with removing this paragraph entirely since it is implied.